android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -144,6 +144,7 @@ private fun String.toTimelineReferencePoint(): TimelineReferencePoint? = when (t
  */
 private fun ReadableMap.toAdaptationConfig(): AdaptationConfig = AdaptationConfig().apply {
     withInt("maxSelectableBitrate") { maxSelectableVideoBitrate = it }
+    withInt("initialBandwidthEstimateOverride") { initialBandwidthEstimateOverride = it.toLong(); }
```

Comment:
JS `number` is a double. Not sure why this is an Int :thinking: Should both be changes to `withDouble`?

Reply:
I'll leave it as-is for now please as this seems inline with the `withInt` implementation of both `bandwidthEstimateWeightLimit` and `maxSelectableBitrate`